pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -31,10 +31,16 @@ import (
 
 type Action struct {
 	session *framework.Session
+	// configured flag for error cache
+	enablePredicateErrorCache bool
+	enableGangCheckOverused   bool // indicates whether consider job's min request(gang-scheduling) when check overused
 }
 
 func New() *Action {
-	return &Action{}
+	return &Action{
+		enablePredicateErrorCache: true,  // default to enable it
+		enableGangCheckOverused:   false, // default to disable it
```

Comment:
Why called `Gangxxx`?

Reply:
It will job's min request resource with gang-scheduling, not just a task's request.